extendr-api/src/graphics/device_descriptor.rs

```diff
@@ -0,0 +1,241 @@
+use super::{color::Color, FontFace, LineType};
+
+// R internals says:
```

Comment:
Perhaps 'states' would be better? Or simply 'From R internals:'
If it is written in the source code, perhaps you can insert a link to R mirror on github?

Reply:
It is.

https://github.com/wch/r-source/blob/trunk/doc/manual/R-ints.texi

But, I think I should link to the HTML version (https://cran.r-project.org/doc/manuals/r-devel/R-ints.html)